styles/layouts/l-FAQ.scss

```diff
@@ -31,6 +31,13 @@
       max-width: $content-max-width;
       margin: 0 0 0 $anchor-list-max-width;
 
+      a {
```

Comment:
This block is duplicated 4 times. You could add for example a `content-link` class to those links and add the style to _texts.scss, or add a class to the body (`.content-page a`)  and add the style to _texts.scss